common/workunit/workunit.cpp

```diff
@@ -5960,11 +5953,6 @@ void CLocalWorkUnit::_loadStatistics() const
 IConstWUStatisticIterator& CLocalWorkUnit::getStatistics(const IStatisticsFilter * filter) const
 {
     CriticalBlock block(crit);
-    //This should be deleted in version 6.0 when support for 4.x is no longer required
-    legacyTimings.loadBranch(p,"Timings");
```

Comment:
I don't think so - it would be very unusual to have a workunit from 4.0 in a 7.0 system.  If they did the only problem would be timings didn't appear any more.  @richardkchapman do we document things like this?